tests/software_tests/translator/test_translator.py

```
@@ -149,34 +146,34 @@ def test_encode__value_error(self, sid, rsid, services_mapping):
 
-    @pytest.mark.parametrize("message", [
-        UdsMessage(payload=[0x10, 0x03], addressing_type=AddressingType.PHYSICAL),
-        Mock(spec=UdsMessageRecord, payload=[0x62, *range(255)])
+    @pytest.mark.parametrize("payload", [
+        [0x10, 0x03],
+        [0x62, *range(255)]
     ])
-    def test_decode__value_error(self, message):
+    def test_decode__value_error(self, payload):
         self.mock_translator.services_mapping = {}
         with pytest.raises(ValueError):
```

Ruff (0.15.20)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)